### ISSUE TITLE
Question: How to escape quotes?

### ISSUE THREAD

**Issue by TSSlade** [softwaretechnik-berlin/dbml-renderer#9]
I get 
```bash
(base) ▸▸▸▸▸▸▸▸▸▸ tsslade 10164 job-hunting/komaza  % → dbml-renderer -i komaza.dbml -o komaza.svg
Could not parse input at line 31. Expected "'" or "'''" but "\"" found.

```
when I use a double-quoted string to define a Table `Note` that contains a possessive (`"Farmer's"`).

So then I changed to all single-quoted notes and attempted to escape the text (`'Famer\'s'`) and got the error message below.
```bash
(base) ▸▸▸▸▸▸▸▸▸▸ tsslade 10166 job-hunting/komaza  % → dbml-renderer -i komaza.dbml -o komaza.svg
Could not parse input at line 54. Expected "," or "]" but "s" found.
```

Same thing if I escape the escape: `'Farmer\\'s'`

Is the dbml-renderer parsing engine unable to handle internal single quotes? Or am I missing something about how to properly handle them?

**Comment by fbartho** [comment]
`@dbml/cli` took my `postgres pg_dump --schema-only` output, and produced a DBML file that has double-quotes around every identifier.

Unfortunately, the renderer exploded on double-quotes, but it was relatively simple to strip them out, as a workaround.

**Comment by barnardb** [comment]
Thanks for reporting. A test case has been added for this issue, and it's been fixed in v1.0.20.
